.github/workflows/github-actions-essential-ci.yml

```diff
@@ -344,3 +344,45 @@ jobs:
       - name: clean up
         run: ./build/github/cleanup-engflow-keys.sh
         if: always()
+  cockroach-microbench-ci:
+    runs-on: [ self-hosted, basic_runner_group ]
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
Please add the following steps to the build, like in all the others:

```
      - name: compute metadata
        run: echo GITHUB_ACTIONS_BRANCH=${{ github.event.pull_request.number || github.ref_name}} >> "$GITHUB_ENV"
      - run: ./build/github/prepare-summarize-build.sh
      - name: upload build results
        run: ./build/github/summarize-build.sh bes.bin
        if: always()
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Check the other builds to see what the order relative to all the others should be.